kcidb/monitor/spool/__init__.py

```diff
@@ -157,14 +175,17 @@ def pick(self, id, timestamp=None, timeout=None):
         def pick_if_not_picked(transaction, doc, timestamp):
             """Pick notification, if not picked yet"""
             # Get the document snapshot
-            snapshot = doc.get(field_paths=["picked_until", "message"],
+            snapshot = doc.get(field_paths=["picked_until", "message",
+                                            "due"],
                                transaction=transaction)
             picked_until = snapshot.get("picked_until")
             message_text = snapshot.get("message")
+            due = snapshot.get('due')
             # If the document doesn't exist, has no "picked_until" field,
             # no "message" field, or the picking has not timed out yet
+            # or the due date has not passed
             if not picked_until or not message_text or \
-               picked_until > timestamp:
+                    picked_until > timestamp or due > timestamp:
```

Comment:
Is this whitespace change necessary?

Reply:
No
I will fix it, sorry for the unnecessary change

Reply:
No problem, thanks :)

Reply:
Could you revert this whitespace change, if it's unnecessary?